test/Microsoft.AspNetCore.Mvc.Razor.Test/RazorViewEngineOptionsTest.cs

```diff
@@ -12,6 +13,65 @@ namespace Microsoft.AspNetCore.Mvc.Razor
     public class RazorViewEngineOptionsTest
     {
         [Fact]
+        public void AreaViewLocationFormats_ContainsExpectedLocations()
+        {
+            // Arrange
+            var services = new ServiceCollection().AddOptions();
+            var areaViewLocations = new[]
+            {
+                "/Areas/{2}/MvcViews/{1}/{0}.cshtml",
+                "/Areas/{2}/MvcViews/Shared/{0}.cshtml",
+                "/MvcViews/Shared/{0}.cshtml"
+            }.AsEnumerable();
+
+            // Act
+            var builder = new MvcBuilder(services, new ApplicationPartManager());
+            builder.AddRazorOptions(options =>
+            {
+                options.AreaViewLocationFormats.Clear();
+
+                foreach (var location in areaViewLocations)
+                {
+                    options.AreaViewLocationFormats.Add(location);
+                }
+            });
+            var serviceProvider = services.BuildServiceProvider();
```

Comment:
What part of this is actually the `// Act`? I suspect the `AddRazorOptions()` call and the rest should be in `// Arrange` or `// Assert`. If so, this is a test of `MvcRazorMvcBuilderExtensions`.